Mathlib/Algebra/Group/ULift.lean

```diff
@@ -143,9 +143,9 @@ instance commMonoid [CommMonoid α] : CommMonoid (ULift α) :=
 #align ulift.comm_monoid ULift.commMonoid
 #align ulift.add_comm_monoid ULift.addCommMonoid
 
-instance natCast [NatCast α] : NatCast (ULift α) := ⟨λ a ↦ up a⟩
+instance natCast [NatCast α] : NatCast (ULift α) := ⟨fun a ↦ up a⟩
 #align ulift.has_nat_cast ULift.natCast
-instance intCast [IntCast α] : IntCast (ULift α) := ⟨λ a ↦ up a⟩
+instance intCast [IntCast α] : IntCast (ULift α) := ⟨fun a ↦ up a⟩
```

Comment:
```suggestion
instance intCast [IntCast α] : IntCast (ULift α) := ⟨(up ·)⟩
```
and on line 185 use `(⟨·⟩)` (note that the parentheses are outside the anonymous constructor)